src/Settings/SettingsServiceProvider.php

```diff
@@ -22,7 +22,10 @@ class SettingsServiceProvider extends AbstractServiceProvider
     public function register()
     {
         $this->container->singleton('flarum.settings.default', function () {
-            return new Collection();
+            return new Collection([
+                'theme_primary_color' => '#4D698E',
+                'theme_secondary_color' => '#4D698E',
+            ]);
```

Comment:
Would we want to actually remove the defaults from the installation step in favor of this ? 🤔

Reply:
That's the goal (as well as getting rid of data migrations). Probably in a separate PR though.